Lesson_2_Python_II/Lesson_2_Clean_v2.ipynb

```diff
@@ -1022,39 +1022,1084 @@
   },
```

Comment:
<p>remove answers from here. This is the clean version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>